src/e2e-test/java/io.cdap.plugin/stepsdesign/StepDefinition.java

```diff
@@ -30,55 +30,17 @@
  * Contains Oracle replication test scenarios step definitions.
  */
 public class StepDefinition implements CdfHelper {
-    @Given("Open DataFusion Project with replication to configure pipeline")
-    public void openDataFusionProjectWithReplicationToConfigurePipeline() throws IOException, InterruptedException {
-        openCdf();
-        SeleniumDriver.getDriver().get(SeleniumDriver.getDriver().getCurrentUrl().replace(
-                PluginPropertyUtils.pluginProp("cdfUrl"), PluginPropertyUtils.pluginProp("replication.url")));
-    }
-    @And("Click on the Next button")
-    public void clickOnTheNextButton() throws InterruptedException {
-        ReplicationActions.clickNextButton();
-    }
-    @And("Select Oracle as Source")
-    public void selectSourceAsOracle() {
-        ReplicationActions.clickOnOraclePlugin();
-    }
 
     @Then("Validate Source table is available and select it")
     public void selectTable() {
         ReplicationActions.selectTable();
     }
-    @Then("Deploy the replication pipeline")
-    public void deployPipeline() {
-        ReplicationActions.deployPipeline();
-    }
-
-    @Then("Run the replication Pipeline")
-    public void runPipelineInRuntime() {
-        ReplicationActions.runThePipeline();
-    }
-
-    @Then("Open the logs")
-    public void openLogsOfSltPipeline() {
-        ReplicationActions.openAdvanceLogs();
-    }
 
     @Then("Wait till pipeline is in running state and check if no errors occurred")
     public void waitTillSltPipelineIsInRunningState() throws InterruptedException {
         ReplicationActions.waitTillPipelineIsRunningAndCheckForErrors();
     }
 
-    @Then("Close the pipeline logs and stop the pipeline")
-    public void closeLogsAndStopThePipeline() {
-        ReplicationActions.closeTheLogsAndClickOnStopButton();
-    }
-
-    @And("Capture raw logs")
-    public void captureRawLogs() {
-        ReplicationActions.captureRawLog();
-    }
-
     @And("Run insert, update and delete CDC events on source table")
```

Comment:
except for this, all are common steps that are used in all sources.Can you place in common stepsdesign?

Reply:
Here, we need the schema name, database name which differ for source plugins table creation and are mentioned and used from pluginParameter file for pipeline initialization. All the other common replication steps are already placed and used from cdap e2e. https://docs.google.com/document/d/1MXdQujYAzmBmZBGxrCjGPRHbAuhzOT3TXR9DOoqLRGA/edit?resourcekey=0-JpDyJhg4BnXXhMJ8fNF7dw